framework/behaviors/BlameableBehavior.php

```diff
@@ -73,6 +73,15 @@ class BlameableBehavior extends AttributeBehavior
      */
     public $value;
 
+    /**
+     * @var mixed Default value for cases when user is guest
+     */
+    public $defaultValue;
+
+    /**
+     * @var bool Whether is [[defaultValue]] enabled
+     */
+    public $useDefaultValue = false;
```

Comment:
This property is redundant. `$defaultValue` is `null` by default, same as `Yii::$app->get('user')->id` for guests. We don't need separate property to decide which `null` should we use.